lib/Platform.cs

```diff
@@ -35,49 +35,15 @@ public enum PlatformName : int
 
 	public static partial class Platform
 	{
-		public static readonly string[] Compilers = new string[] {
-			"msvc2008",
-			"msvc2010",
-			"msvc2012",
-			"msvc2013",
-			"msvc2015",
-			"gcc3",
-			"gcc4",
-			"gcc5",
-			"mingw32",
-		};
 
-		// public static readonly string LibraryName;
-
-		// public static readonly string LibraryFileExtension;
-
-		public static readonly string[] LibraryPaths;
-
-		public delegate UnmanagedLibrary LoadUnmanagedLibraryDelegate(string libraryName);
-		public static readonly LoadUnmanagedLibraryDelegate LoadUnmanagedLibrary;
-
-		public delegate SafeLibraryHandle OpenHandleDelegate(string filename);
-		public static readonly OpenHandleDelegate OpenHandle;
-
-		public delegate IntPtr LoadProcedureDelegate(SafeLibraryHandle handle, string functionName);
-		public static readonly LoadProcedureDelegate LoadProcedure;
-
-		public delegate bool ReleaseHandleDelegate(IntPtr handle);
-		public static readonly ReleaseHandleDelegate ReleaseHandle;
-
-		public delegate Exception GetLastLibraryErrorDelegate();
-		public static readonly GetLastLibraryErrorDelegate GetLastLibraryError;
-
-		public static readonly bool Is__Internal;
+		// public static readonly bool Is__Internal;
```

Comment:
Should probably be removed if no longer needed.

Reply:
Yes and no... I didn't test this on my own, however Android builds were using __Internal instead of libzmq as DllImport. I don't know currently, maybe we should really use a define directive or the like, if you are on Android.

Reply:
Ok... Then leave it for the moment. 

Perhaps we should add a travis build on Android?

Reply:
Yes, I would also like a Windows build on travis-ci...

Reply:
We already have Windows CI, but on appveyor... travis-ci does not support Windows.